modules/homebrew.nix

```diff
@@ -789,7 +785,8 @@ in
       # Homebrew Bundle
       echo >&2 "Homebrew bundle..."
       if [ -f "${cfg.brewPrefix}/brew" ]; then
```

Comment:
I don’t think this is very likely, but what if `brew` is not in `brewPrefix`[^1]? Then `mas` won’t be added to the `PATH`.

[^1]: If `brew` isn’t on the `PATH`, then there are bigger issues, but it could just be in a non-standard location, but still on the `PATH`.

Reply:
The `else` condition is just an error message, so that seems okay?

Reply:
Whoops – I overlooked that. Yeah, totally fine, sorry.